### PR TITLE
feat: tutor-mfe compatiblilty for `atlas pull` | FC-0012

### PR DESCRIPTION
**Pull request by OmarIthawi**
Similar to https://github.com/openedx/frontend-app-communications/pull/187.

### Changes
 - install atlas
 - remove `--filter` to pull all languages by default
 - use ATLAS_OPTIONS to allow custom `--filter`
 - include frontend-lib-special-exams in `atlas pull`

## Refs
This pull request is part of the [FC-0012 project](https://openedx.atlassian.net/l/cp/XGS0iCcQ) which implements the [Translation Infrastructure update OEP-58](https://docs.openedx.org/en/latest/developers/concepts/oep58.html).

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @OmarIthawi! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: Makefile [hunk 1]
```diff
@@ -53,13 +53,15 @@ pull_translations:
 	rm -rf src/i18n/messages
 	mkdir src/i18n/messages
 	cd src/i18n/messages \
-      && atlas pull --filter=$(transifex_langs) \
+      && atlas pull $(ATLAS_OPTIONS) \
+               translations/frontend-platform/src/i18n/messages:frontend-platform \
                translations/paragon/src/i18n/messages:paragon \
                translations/frontend-component-header/src/i18n/messages:frontend-component-header \
                translations/frontend-component-footer/src/i18n/messages:frontend-component-footer \
+			   translations/frontend-lib-special-exams/src/i18n/messages:frontend-lib-special-exams \
```
Comment (brian-smith-tcril) [inline Makefile]: nit: formatting (mixed tabs/spaces)

```suggestion
               translations/frontend-lib-special-exams/src/i18n/messages:frontend-lib-special-exams \
```

**Review by brian-smith-tcril — APPROVED** [review]
tiny nit but overall LGTM!

File: Makefile (hunk 1 above)
Comment (OmarIthawi) [reply Makefile]: Good catch. My editor fails me occasionally on mixed spacing types.

Now fixed.

**Comment by brian-smith-tcril** [comment]
@mattcarter you are listed as "Please inform" for this repo on the spreadsheet, I plan on merging this by EOD today, please let me know if I should hold off.

**Comment by mattcarter** [comment]
> @mattcarter you are listed as "Please inform" for this repo on the spreadsheet, I plan on merging this by EOD today, please let me know if I should hold off.

Hi @brian-smith-tcril, I'm no longer a 2U employee. I suggest reaching out to @schenedx. Thanks!

**Comment by openedx-webhooks** [comment]
@OmarIthawi 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-learning&entry.867055334=https://github.com/openedx/frontend-app-learning/pull/1279&entry.1484655318=https://github.com/OmarIthawi&entry.752974735=2024-01-29+14:06&entry.1917517419=2024-01-29+17:40&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**